taskcluster/translations_taskgraph/actions/train.py

```diff
@@ -143,6 +143,13 @@ def validate_pretrained_models(params):
                     "use-opuscleaner": {
                         "type": "string",
                         "description": "use OpusCleaner to clean corpus",
+                        "enum": ["true", "false"],
+                    },
+                    "opuscleaner-mode": {
```

Comment:
Thanks for remembering to update this & parameters.py.

Reply:
sure, I filed #597 though :)